models/comments.model.js

```diff
@@ -0,0 +1,16 @@
+const db = require("../db/connection");
+
+exports.insertComments = (comment, article_id) => {
+  const queryString =
+    "INSERT INTO comments (author, body, article_id) VALUES($1, $2, $3) RETURNING *";
+
+  const queryValues = Object.values(comment);
```

Comment:
Object.values isn't the best way to create the array as the order of the elements cannot 100% be guaranteed.Simply taking the required properties of the comment object would be safer and also make the first extra test I mention pass too 👍